packages/common/src/hooks/useQueryParams/useQueryParams.test.tsx

```diff
@@ -15,6 +15,7 @@ type ThemeChangererProps = {
   dataRowHeight: number;
   headerRowHeight: number;
   footerHeight: number;
+  saveButtonRowHeight: number;
```

Comment:
I think this is a bit of a problem .. at the moment i've not adjusted further for the difference between list and detail views (not have the save button row).

I also find managing these rows to just generally be an annoying to handle, constantly update and the weird ui jankiness it causes is a bit horrible. Maybe there's a better solution for this,  but I am not sure anymore what we are really gaining by not just scrolling the rows in the table 🤷

Reply:
When doing this sort of thing previously, I've ended up scrolling in some way. There's a lot you can do with determining height and being tricksy, but there are so many edge cases and the whole thing ends up being so fragile, just doesn't seem worth it.

that said, I think stick with what we have for now - there's other stuff to tidy up first :)